contrib/confluence/src/python/pants/contrib/confluence/tasks/BUILD

```diff
@@ -0,0 +1,15 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+python_library(
+  dependencies = [
+    'contrib/confluence/src/python/pants/contrib/confluence/targets',
+    'contrib/confluence/src/python/pants/contrib/confluence/util',
+    'src/python/pants/base:exceptions',
+    'src/python/pants/task',
+    'src/python/pants/util:desktop',
+    'src/python/pants/util:dirutil',
+  ],
+  sources=['confluence_publish.py'],
```

Comment:
Should drop the sources glob to let this target use the default glob (`*.py`) and own the whole directory.